spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -327,7 +327,16 @@
     end
 
     context 'when the user goes through reproofing' do
-      let!(:user) { create(:user, profiles: [create(:profile, :active)]) }
+      # TODO: this passed with the previous `active` trait,
+      # but now fails with the new one that removes `activated_at`
+      # let!(:user) { create(:user, profiles: [create(:profile, :active)]) }
+
+      # this will pass with the new `deactivated` trait, but it obviously shouldn't
+      # let!(:user) { create(:user, profiles: [create(:profile, :deactivated)]) }
+
+      # TODO: this preserves the previous passing behavior, but should be re-implemented
+      # to rely on the `Profile#active` boolean rather than the `Profile#activated_at` timestamp
+      let!(:user) { create(:user, profiles: [create(:profile, activated_at: Time.zone.now)]) }
```

Comment:
note: Here is the spec below that fails:
https://github.com/18F/identity-idp/pull/8629/commits/e08d8f8f3e0e970b1cb1fa1d9569fb644fd3d3af#diff-6bf8b5abaee036aff5e33c715c093fd644b2a81cd370915848679e2914ce8fdeR365